.github/workflows/build-workflow.yml

```diff
@@ -59,10 +59,6 @@ jobs:
         with:
           targets: ${{ matrix.target }}
 
-      - name: Enable cache
-        # https://github.com/marketplace/actions/rust-cache
-        uses: Swatinem/rust-cache@v2
-
```

Comment:
This may be a workaround for now. I feel its better to create an issue in the `rust-cache`.